docs/xmlprofiles.rst

```diff
@@ -234,25 +234,22 @@ Member types
 
 Member types are any type that can belong to a ``<struct>`` or a ``<union>``, or be aliased by a ``<typedef>``.
 
-When used as ``<sequence>``'s elements, ``key`` or ``value`` types of a map, as an aliased type, etc.,
-its ``name`` attribute is ignored and can be omitted.
-
 **Basic types**
 
 The tags of the available basic types are:
 
 +--------------------------+--------------------------+--------------------------+
-| ``<boolean>``            | ``<longlong>``           | ``<longdouble>``         |
+| ``<boolean>``            | ``<int64>``              | ``<float64>``            |
 +--------------------------+--------------------------+--------------------------+
-| ``<octet>``              | ``<unsignedshort>``      | ``<string>``             |
+| ``<byte>``               | ``<uint16>``             | ``<string>``             |
 +--------------------------+--------------------------+--------------------------+
-| ``<char>``               | ``<unsignedlong>``       | ``<wstring>``            |
+| ``<char>``               | ``<uint32>``             | ``<wstring>``            |
 +--------------------------+--------------------------+--------------------------+
-| ``<wchar>``              | ``<unsignedlonglong>``   | ``<boundedString>``      |
+| ``<wchar>``              | ``<uint64>``             |                          |
 +--------------------------+--------------------------+--------------------------+
-| ``<short>``              | ``<float>``              | ``<boundedWString>``     |
+| ``<int16>``              | ``<float32>``            |                          |
 +--------------------------+--------------------------+--------------------------+
-| ``<long>``               | ``<double>``             |                          |
+| ``<int32>``              | ``<float64>``            |                          |
```

Comment:
This section must be updated to reflect that now these keywords are `type` attributes instead tags. And remember to remove the tag symbols (`<` and `>`).

Reply:
Done